lib/bootstrap.js

```diff
@@ -133,11 +141,42 @@ module.exports = function(config) {
             }
           }))
           .pipe(template(templateConfig))
-          .pipe(conflict(rootDir))
+          .pipe(conflict(rootDir, {
+            logger: function(message, fileName, extraText) {
+              console.log(chalk.green('✓ ') + message + ' ' + chalk.stripColor(fileName));
+            },
+          }))
           .pipe(gulp.dest(rootDir))
-          .pipe(install())
+          .on('end', function() {
+            console.log('\n' + chalk.green('✓ ') + 'Creating files… done!');
+            cli.spinner('  Installing npm dependencies…');
+          })
+          .pipe(install({
+            log: function() {
+              cli.spinner(chalk.red('× ') + 'Installing npm dependencies… error!\n', true);
+              console.log(Array.prototype.slice.call(arguments).join(''));
+            },
+            npmStdio: ['ignore', 'ignore', 'ignore'],
+          }))
+          .on('end', function() {
+            cli.spinner(chalk.green('✓ ') + 'Installing npm dependencies… done!\n', true);
+          })
           .pipe(sink()); // Sink is required to trigger the finish event with install.
-        stream.on('finish', resolve);
+
+        stream.on('finish', function() {
+          console.log(
+            'Your app has been bootstrapped! Just commit the changes and push the commit\n' +
+            'to the origin/master branch:\n\n' +
+            chalk.bold('git commit -m"initial version of Oghliner app" --all') + '\n' +
```

Comment:
Per that issue @wfwalker ran into with *configure*, I guess this should be `git add --all && git commit -m"initial version of Oghliner app"`.